schema/models_directory/dft.json

```diff
@@ -10,17 +10,33 @@
     ],
     "properties": {
         "type": {
-            "enum": [
-                "dft"
-            ]
+            "properties": {
+                "name": {
+                    "type": "string"
```

Comment:
Has to be:

`{
    "enum": [ 
                        "Density Functional Theory"
                    ]
}`